apps-rendering/src/pagination.ts

```diff
@@ -0,0 +1,146 @@
+import { map, none, some, withDefault } from '@guardian/types';
+import type { Option } from '@guardian/types';
+import { pipe } from 'lib';
+import type { LiveBlock } from 'liveBlock';
+
+export type PageReference = {
+	blocks: LiveBlock[];
+	pageNumber: number;
+	suffix: string;
+};
+
+export type Pagination = {
+	newest: Option<string>;
+	newer: Option<string>;
+	oldest: Option<string>;
+	older: Option<string>;
+	numberOfPages: number;
+};
+
+export type LiveBlogPagedBlocks = {
+	currentPage: PageReference;
+	pagination: Pagination;
+};
+
+const navContainerId = 'liveblog-navigation';
```

Comment:
I am tempted to remove this from pagination.ts and possibly include it in the to-be-implemented `Pagination` component. All we're trying to achieve here is scrolling to the pagination navigation container when going to any page (other than the first page). I wonder if it seems strange to reference a container id here, when possibly it's more related to the `Pagination` component itself (which will include the navigation container and therefore define this id).

Reply:
I removed this as it didn't 'feel' right - given the 'scroll to pagination navigation' feature isn't yet working in DCR, it might be that we can support this in the Pagination component itself (providing support for both AR+DCR at the same time)